lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -267,15 +302,26 @@ def self.bump_version_release
           return verified_version
         end
 
-        # Update the `build.gradle` file with new `versionName` and `versionCode` values, both or the `defaultConfig` and `vanilla` flavors
+        # Update the `version.properties` file with new `versionName` and `versionCode` values
         #
-        # @param [Hash] new_version_beta The version hash for the beta (vanilla flavor), containing values for keys "name" and "code"
-        # @param [Hash] new_version_alpha The version hash for the alpha (defaultConfig), containing values for keys "name" and "code"
-        # @env HAS_ALPHA_VERSION If set (with any value), indicates that the project uses `vanilla` flavor.
+        # @param [Hash] new_version_beta The version hash for the beta, containing values for keys "name" and "code"
+        # @param [Hash] new_version_alpha The version hash for the alpha , containing values for keys "name" and "code"
         #
-        def self.update_versions(new_version_beta, new_version_alpha)
-          self.update_version(new_version_beta, ENV['HAS_ALPHA_VERSION'].nil? ? 'defaultConfig' : 'vanilla {')
-          self.update_version(new_version_alpha, 'defaultConfig') unless new_version_alpha.nil?
+        def self.update_versions(app, new_version_beta, new_version_alpha)
```

Comment:
Missing doc for `@param [String] app`

TIP: you can use `bundle exec rake docstats` to find missing documentation, and `bundle exec rake docstats[subdir/path]` to run the stats only on a given sub-directory (like the ones where you made changes in that PR, to avoid noise about existing code you didn't touch)

Reply:
TIL

Reply:
Resolved in https://github.com/wordpress-mobile/release-toolkit/pull/258/commits/97b8b032190c097de9b5807e4f9c09c254b1a2f3